framework/PostProcessors/ETImporter.py

```diff
@@ -209,7 +209,7 @@ def checkETstructure(self,links,listETs,connectivityMatrix):
     if len(zeroColumns)==0:
       self.raiseAnError(IOError, 'No root ET')
     if len(zeroColumns)==1:
-      rootETID = listETs[zeroColumns]
+      rootETID = listETs[zeroColumns[0]]
```

Comment:
why this change?

Reply:
The ```zeroColumns``` is a 1-D array, and we can not directly use it in the list.